setup.cfg

```diff
@@ -1,4 +1,47 @@
-[egg_info]
-tag_build = .dev0
-#tag_svn_revision = yes
-#tag_date = yes
+[metadata]
+name = dill
+author = Mike McKerns
+maintainer = Mike McKerns
+license = BSD-3-Clause
+license_file = LICENSE
+description = serialize all of python
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://pypi.org/project/dill
+homepage = https://github.com/uqfoundation/dill
+download_url = https://github.com/uqfoundation/dill
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.5
+    Programming Language :: Python :: 2.6
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.1
+    Programming Language :: Python :: 3.2
+    Programming Language :: Python :: 3.3
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Topic :: Scientific/Engineering
+    Topic :: Software Development
+platforms = Linux, Windows, Mac
+
+[options]
+packages = dill, dill.tools
```

Comment:
I had `dill.tests` as a module.  While it might not be 100% desirable to include `tests` as a module, it at least was necessary for tests to run in some cases.  Needs investigation.